app/views/declaration/declaration_holder.scala.html

```diff
@@ -20,62 +20,121 @@
 @import services.HolderOfAuthorisationCode
 @import services.view.AutoCompleteItem
 @import views.components.fields.LabelledValue
-@import views.Title
-@import controllers.navigation.Navigator
 @import models.requests.JourneyRequest
 @import models.Mode._
+@import controllers.navigation.Navigator
+@import models.requests.JourneyRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.{BackButton, Title}
+@import views.html.components.gds._
+@import play.twirl.api.HtmlFormat
+@import scala.collection.immutable
 
-@this(main_template: views.html.main_template)
+@this(
+        govukLayout: gdsMainTemplate,
+        govukButton: GovukButton,
+        govukTable : GovukTable,
+        errorSummary: errorSummary,
+        sectionHeader: sectionHeader,
+        pageTitle: pageTitle,
+        exportsInputCheckbox: exportsInputCheckbox,
+        exportsInputText: exportsInputText,
+        tariffDetails: tariffDetails,
+        saveButtons: saveButtons,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(mode: Mode, form: Form[DeclarationHolder], holders: Seq[DeclarationHolder])(implicit request: JourneyRequest[_], messages: Messages)
 
 @tariffBody = { @Html(messages("declaration.type.holderOfAuthorisationSummaryText", components.details_content_link())) }
 
-@main_template(
-    title = Title("supplementary.declarationHolder.title", "supplementary.summary.parties.header")
-) {
+@addButton = @{
+govukButton(Button(
+    classes = "govuk-button--secondary",
+    content = HtmlContent(HtmlFormat.fill(
+        immutable.Seq(
+            Html(s"""<span aria-hidden="true">${messages("site.add")}</span>"""),
+            Html(s"""<span class="govuk-visually-hidden">${messages("supplementary.additionalActors.add.hint")}</span>""")
+        )
+    )),
+    name = Some("Add"),
+    attributes = Map("id" -> "add")
+))
+}
 
-    @components.back_link(Navigator.backLink(DeclarationHolder, mode), mode)
+@removeButton(value: Option[String]) = @{
```

Comment:
We have a `removeButton` partial, please use that one ;)

Reply:
See procedure_codes view